README.md

```diff
@@ -1,6 +1,6 @@
 # Helm
 
-[![CircleCI](https://circleci.com/gh/helm/helm.svg?style=shield)](https://circleci.com/gh/helm/helm)
+[![Build Status](https://github.com/helm/helm/workflows/push/badge.svg?branch=main)](https://github.com/helm/helm/actions?workflow=build-pr)
```

Comment:
I think we want the release badge for the Readme, right? According to GH UI, I believe this is the right syntax (if no branch query is specified, the default branch - main - is assumed)

```suggestion
[![Build Status](https://github.com/helm/helm/workflows/release.yml/badge.svg)](https://github.com/helm/helm/actions/workflows/release.yml)
```